src/github-connector.ts

```diff
@@ -25,31 +25,41 @@ export class GithubConnector {
     return this.githubData.pullRequest.head.ref;
   }
 
-  getIssueKeyFromTitle(): string {
+  getIssueKeyFromTitle(): { key: string; source: ESource } {
     const { WHAT_TO_USE } = getInputs();
 
     const prTitle = this.githubData.pullRequest.title || '';
     const branchName = this.headBranch;
 
     let keyFound: string | null = null;
+    let source: ESource | null = null;
 
     switch (WHAT_TO_USE) {
       case ESource.branch:
         keyFound = this.getIssueKeyFromString(branchName);
+        source = ESource.branch;
         break;
       case ESource.prTitle:
         keyFound = this.getIssueKeyFromString(prTitle);
+        source = ESource.prTitle;
```

Comment:
hi, to keep consistency with `both` option, you should add here (and above)
```
 source = keyFround ? ESource.prTitle : null;
```

Reply:
Thank you! Done